Spokestack/SpeechConfiguration.swift

```diff
@@ -125,4 +125,6 @@ import Foundation
     @objc public var tracing: Trace.Level = Trace.Level.NONE
     /// Delegate events will be sent using the specified dispatch queue.
     @objc public var delegateDispatchQueue: DispatchQueue = DispatchQueue.global(qos: .userInitiated)
+    /// The dynamic size of the buffer in use by the `AudioEngine`.
+    internal var audioEngineBufferSize: UInt32 = 320
```

Comment:
You do document it as dynamic here, but shouldn't the actual value be dependent on the other config variables?

Reply:
no, it floats based on what the input format is. i don't need it any specific size where it's being used, it just needs to match across AudioEngine/AUUnits.

Reply:
Huh, OK. Apple 🤷